language/stdlib/modules/LibraTransactionPublishingOption.move

```diff
@@ -82,6 +98,12 @@ module LibraTransactionPublishingOption {
         publish_option.script_allow_list = Vector::empty();
         LibraConfig::set<LibraTransactionPublishingOption>(lr_account, publish_option);
     }
+    spec fun set_open_script {
+        pragma aborts_if_is_partial = true;
+        include Roles::AbortsIfNotLibraRoot{account: lr_account}; // Aborts if not LibraRoot [B20].
+        aborts_with Errors::INVALID_ARGUMENT, Errors::REQUIRES_CAPABILITY, Errors::NOT_PUBLISHED;
+        // TODO: this spec block is incomplete.
```

Comment:
I would put this at the beginning of the spec block (here and elsewhere). Also why not use `///`?